src/Commands/AddCommand.php

```diff
@@ -57,7 +59,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
                 file_put_contents($filename, $script);
                 chmod($filename, 0755);
-                $output->writeln("Added <info>{$hook}</info> hook");
+                $output->writeln($fileExists
+                    ? "<info>{$hook}</info> overridden"
```

Comment:
I would prefer `"Added <info>{$hook}</info> hook forcefully"`. The forceful part can even be emphasized with another color. Probably `<fg=red>` or something like that

Reply:
I would prefer to avoid false positive tests between this and:
https://github.com/BrainMaestro/composer-git-hooks/blob/e4b55a716a95777164912d5bfac5f4cdd54c2277/tests/AddCommandTest.php#L26-L33
Because the asserted string would be a subpart of the other.

Reply:
True. Okay, I would still like to maintain the format, so maybe we can use **Overwrote** instead of Added. I don't like the verb though.

Reply:
> I don't like the verb though.

LoL, why? :)